deprecations/index.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * This object's keys are (semver) version numbers, and the
+ * values are arrays of objects each with a "selectors"
+ * array and a "message" string.
+ */
+const versionDeprecations = {
+  '13.0.0': [
```

Comment:
I wonder if it'd be better to store this object in a json file 🤷‍♂ I'm not sure we'd get a ton of benefit from that but I kinda like the separating the serializable deprecations data from the functions. I don't feel too strongly about this though.

Reply:
Oh, looks like you're using this file to build a json file. That's probably fine 👍